semantic-ui-react-redux/src/main/scala/demo/Demo.scala

```diff
@@ -0,0 +1,55 @@
+package demo
+
+import demo.ReduxFacade.Connected
+import japgolly.scalajs.react.raw.React
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.{CallbackTo, ScalaFnComponent}
+import typings.semanticUiReact.anon.MenuItem
+import typings.semanticUiReact.cardCardMod.CardProps
+import typings.semanticUiReact.headerHeaderMod.HeaderProps
+import typings.semanticUiReact.progressProgressMod.ProgressProps
+import typings.semanticUiReact.searchSearchMod.SearchProps
+import typings.semanticUiReact.semanticUiReactStrings.large
+import typings.semanticUiReact.tabTabMod.TabProps
+import typings.semanticUiReact.{semanticUiReactStrings, components => Sui}
+
+import scala.scalajs.js
+
+object Demo {
+
+  val CardDemo: React.Element =
+    Sui.Card.withProps(CardProps().setColor(semanticUiReactStrings.orange))(
```

Comment:
See if you can make these more like this:
```scala
Sui.Card.color(semanticUiReactStrings.orange)
```
Same for all usages of `withProps` further down